llvm/lib/Target/AMDGPU/AMDGPULegalizerInfo.cpp

```diff
@@ -4723,10 +4722,23 @@ bool AMDGPULegalizerInfo::legalizeFDIV32(MachineInstr &MI,
                        .setMIFlags(Flags);
   auto NegDivScale0 = B.buildFNeg(S32, DenominatorScaled, Flags);
 
+  const bool PreservesDenormals = Mode.FP32Denormals == DenormalMode::getIEEE();
+  const bool HasDynamicDenormals =
+      (Mode.FP32Denormals.Input == DenormalMode::Dynamic) ||
+      (Mode.FP32Denormals.Output == DenormalMode::Dynamic);
+
   // FIXME: Doesn't correctly model the FP mode switch, and the FP operations
```

Comment:
Still has the fixme?

Reply:
Ah I thought it was unrelated, i'll remove it.